client/containers/DashboardOverview/overviewRows/labels.tsx

```diff
@@ -78,3 +84,101 @@ export const getPubReleasedStateLabel = (pub: Pub) => {
 		icon: 'lock' as const,
 	};
 };
+
+export const getSubmissionStatusLabel = (
+	submissionStatus: Partial<SubmissionStatus>,
+): IconLabelPair => {
+	if (submissionStatus === 'accepted') {
+		return {
+			label: 'Accepted',
+			icon: 'symbol-square',
+			intent: 'success',
+		};
+	}
+	if (submissionStatus === 'declined') {
+		return {
+			label: 'Declined',
+			icon: 'symbol-square',
+			intent: 'danger',
+		};
+	}
+	if (submissionStatus === 'pending') {
+		return {
+			label: 'Pending',
+			icon: 'symbol-square',
+			intent: 'warning',
+		};
+	}
+	return {
+		label: 'Incomplete',
+		icon: 'symbol-square',
+		intent: 'none',
+	};
+};
+
+export const getSubmissionTimeLabel = (pub: Pub) => {
+	const submissiondDate = getPubSubmissionDate(pub);
+	if (submissiondDate) {
+		return {
+			label: (
+				<>
+					<em>Submitted</em>&nbsp;{getDateLabelPart(submissiondDate)}
+				</>
+			),
+			icon: 'time' as const,
+		};
+	}
+	return {
+		label: 'Update Status',
```

Comment:
What does this mean? If there's no submission date yet, we should just not render anything.

Reply:
That makes sense, for some reason I was thinking a pub without a submission status still needs to be rendered